common/components/register_model/spec.yaml

```diff
@@ -0,0 +1,63 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: register_model
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: Register model
+description:
+  Register a model to a workspace or a registry. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
+
+environment: azureml:python-sdk-v2:0.0.1
```

Comment:
Since auto-versioning if 1-up numbered:
```suggestion
environment: azureml:python-sdk-v2:1
```

Reply:
@elesel coming back to below question again: 
![image](https://user-images.githubusercontent.com/61145377/221869988-57e34d0f-b5c9-469d-8544-6a9b13d7aeee.png)

How do we know what env version to use when registering component, if we use auto as env version? Also, what will be the flow of env fixes that will be applied? Will vulnerable env be deprecated and components are required to be registered with newer env version? How will this flow look like?

Reply:
As environments are updated/patched, their versions will increment. Older versions of environments will always be accessible. We need a better way to dynamically update these references, but since components point to specific environment versions in GH and ADO right now, they'd need to be manually updated to use the latest version of the environment.

Reply:
updated